docs/CMakeLists.txt

```diff
@@ -3,6 +3,6 @@ project(docs)
 
 include(${CMAKE_CURRENT_SOURCE_DIR}/../cmake/ClioVersion.cmake)
 
-# Generate `docs` target for doxygen documentation
-# Note: use `cmake --build . --target docs` from your `build` directory to generate the documentation
+# Generate `docs` target for doxygen documentation Note: use `cmake --build . --target docs` from your `build` directory
```

Comment:
This is a bad change imo.

Reply:
I agree, was able to disable it using `cmake-format: off` and `cmake-format: on`